app/components/post/post.js

```diff
@@ -3,14 +3,7 @@
 
 import React, {PureComponent} from 'react';
 import PropTypes from 'prop-types';
-import {
-    Alert,
-    Clipboard,
-    Platform,
-    TouchableHighlight,
-    View,
-    ViewPropTypes,
-} from 'react-native';
+import {Alert, Clipboard, Platform, TouchableHighlight, View, ViewPropTypes} from 'react-native';
```

Comment:
Not that it matters much, but can you keep this with the previous formatting? I think is easier to read